### PR TITLE
Controlling latency makes data normally distributed in the time dimension

### PR DESCRIPTION
**Pull request by sunzhuohang**
### What problem does this PR solve? 
Add a function to keep the load running, which is a periodic normal distribution in the time dimension

### What is changed and how it works?
Add new parameters in prop.go.
normal_data_in_time : 
Control this function, the default value is false,which means to ignore this function. If the value is true, the go-ycsb will not stop and you need to kill process if you want to stop it.
expectedvalue :
Mean of normal distribution.
standarddeviation :
Standard deviation of normal distribution.
timedelay = 10000 :
Qps decreases with increasing latency.
timeperiod = 20 :
One cycle duration.

### Check List 

Tests 

 - Manual test 
Use this workload file - workload_normal.
QPS and TIKV CPU USAGE are as follows：
image

image

### CODE REVIEW

File: pkg/client/client.go [hunk 1]
```diff
@@ -63,16 +66,18 @@ func newWorker(p *properties.Properties, threadID int, threadCount int, workload
...
-		fmt.Printf("totalOpCount(%s/%s/%s): %d should be bigger than threadCount: %d",
-			prop.OperationCount,
-			prop.InsertCount,
-			prop.RecordCount,
-			totalOpCount,
-			threadCount)
-
-		os.Exit(-1)
+	//Keep data running
+	if !p.GetBool(prop.NormalDataInTime, prop.NormalDataInTimeDefault) {
```
Comment (mahjonp) [inline pkg/client/client.go:70]: Why should keep it running forever? IMO, if you must let it running forever, you can change totalOpCount, it's better than adding a strange if predication statement here.

File: pkg/client/client.go [hunk 2]
```diff
@@ -107,13 +112,17 @@ func (w *worker) throttle(ctx context.Context, startTime time.Time) {
...
 
-func (w *worker) run(ctx context.Context) {
+func (w *worker) run(ctx context.Context,loadStartTime time.Time) {
 	// spread the thread operation out so they don't all hit the DB at the same time
 	if w.targetOpsPerMs > 0.0 && w.targetOpsPerMs <= 1.0 {
 		time.Sleep(time.Duration(rand.Int63n(w.targetOpsTickNs)))
 	}
 
 	startTime := time.Now()
+	mean := w.p.GetFloat64(prop.ExpectedValue,prop.ExpectedValueDefault)
```
Comment (mahjonp) [inline pkg/client/client.go:122]: Seems you embed uncommon logic into here. do we have a better way to do it?

**Comment by mahjonp** [comment]
Please format code and could you explain what's this workload aim?

**Comment by sunzhuohang** [comment]
> Please format code and could you explain what's this workload aim?

To generate a regular load for testing, but it seems to be different from the purpose of using go-ycsb to test database performance.

**Pull request closed without merging**